ipapython/ipaldap.py

```diff
@@ -1289,6 +1291,8 @@ def make_filter_from_attr(
             ]
             return cls.combine_filters(flts, rules)
         elif value is not None:
+            if isinstance(value, IPACertificate):
```

Comment:
I just noticed https://pagure.io/freeipa/issue/7775 and remember that this PR uses IPACertificate. Please use cryptography's certificate class here. It's more generic:

```
>>> from ipalib.x509 import IPACertificate
>>> from cryptography.x509 import Certificate
>>> issubclass(IPACertificate, Certificate)
True
```

Reply:
Hi @tiran
Thanks for the suggestion, implemented in the updated PR.